src/platform/packages/shared/kbn-scout/src/cli/start_server.ts

```diff
@@ -20,11 +31,6 @@ export const startServer: Command<void> = {
   description: 'Start Elasticsearch & Kibana for testing purposes',
```

Comment:
Originally I was thinking to unify all "command" functions, so that we don't use a mix of `runTests` + `runTestFn` but smth like:

```ts
export const startServerCmd: Command<void> = {
} => {
    await startServer();
}
```

It will help to explicitly define Scout CLI commands that we will for sure extend in the future with more features/commands. WDYT?

Reply:
I've added the `Cmd` suffix to all command function names in [d624641](https://github.com/elastic/kibana/pull/216272/commits/d624641961b2152937b56a32dcc937291a2825b3).